packages/addons/service/ttyd/package.mk

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="ttyd"
+PKG_VERSION="1.4.4"
+PKG_SHA256="b910a33ddaa474c369991ba345187a8a2f4aa420389083671ba3a6c305a491d6"
+PKG_REV="100"
+PKG_ARCH="any"
+PKG_LICENSE="MIT"
+PKG_SITE="https://github.com/tsl0922/ttyd"
+PKG_URL="https://github.com/tsl0922/ttyd/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain libwebsockets"
+PKG_SECTION="service"
+PKG_SHORTDESC="ttyd: a tool for accessing the terminal over a browser"
+PKG_LONGDESC="ttyd ($PKG_VERSION_DATE): is a simple command-line tool for sharing terminal over the web"
```

Comment:
Where is PKG_VERSION_DATE being set? Why is this date relevant in the description?